scanners/typo3scan/scanner/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2021 iteratec GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+
+#Base Image
+FROM python:3.9-alpine as base
+
+#Install git and Clone Repo 
+RUN apk add git \
+    && git clone https://github.com/whoot/Typo3Scan.git \
+    && cd Typo3Scan \
+    && rm -r .git .github doc
+COPY wrapper.sh /Typo3Scan/wrapper.sh
+
+#Runtime Image
+FROM python:3.9-alpine as runtime
```

Comment:
Why are two different images necessary (base and runtime)? Seems like the runtime image just copies a directory from the base image and adds privileges and the result directory, should be doable in one image..

Reply:
Okay I think that's a better way than just deleting unnecessary installations and directories :+1: